department-of-veterans-affairs/caseflow
# Judge check out | Evaluate decision

Resolves #5151 

### Description
After a judge reviews a decision and confirms issue dispositions, they are required to evaluate the quality of the attorney's decision for their performance evaluation.

### Acceptance Criteria 
- [ ] Calendar days worked = Decision submitted - Date assigned
- [ ] Default for complexity / case quality is unselected
- [ ] Alert appears if the user chooses 1 or 2 case quality
- [ ] Error state appears when user does not select required fields (complexity, quality, additional factors occasionally)

### Testing Plan
1. Go to `/queue` as Judge (`BVAAABSHIRE`)
2. Select 'Ready for Dispatch' from Action dropdown for a case, verify issue dispositions/remand reasons, confirm Evaluate Decision page appears next
3. After filling in required fields in Evaluate Decision page, confirm Submit Decision page is next

#### Screenshots
<details><summary>No error</summary>
<img src="https://user-images.githubusercontent.com/8533004/41481188-0e04f0da-709f-11e8-8e5b-9dc5623dea94.png">
</details>

<details><summary>Error 1</summary>
<img src="https://user-images.githubusercontent.com/8533004/41481187-0dec01ce-709f-11e8-9019-c9bc79a4416c.png">
</details>

<details><summary>Error 2</summary>
<img src="https://user-images.githubusercontent.com/8533004/41481186-0dd5267a-709f-11e8-9007-1af1e0499b88.png">
</details>

### TODO
- Hook up to backend
- Set [decision type](https://github.com/department-of-veterans-affairs/caseflow/blob/master/client/app/queue/SubmitDecisionView.jsx#L107) earlier
